packages/data-point/lib/helpers/helpers.js

```diff
@@ -68,11 +69,3 @@ function createResolveTransform (dataPoint) {
 }
 
 module.exports.createResolveTransform = createResolveTransform
-
-function isTransform (transform) {
-  return (
-    _.isArray(transform.reducers) && transform.typeOf === 'TransformExpression'
-  )
-}
-
-module.exports.isTransform = isTransform
```

Comment:
Removing this function is a breaking change because it was being exported from `packages/data-point/lib/index.js`

Reply:
is there an equivalent? can we still export it, people creating new entities would find it very useful, if there is such pls add it if not then pls disregard

Reply:
Ok, I added the `isTransform` function from the comment above (along with some tests for it)